tests/js/spec/views/dataExport/dataDownload.spec.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {mountWithTheme} from 'sentry-test/enzyme';
+
+import DataDownload, {DownloadStatus} from 'app/views/dataExport/dataDownload';
+
+describe('DataDownload', function() {
+  beforeEach(MockApiClient.clearMockResponses);
+  const dateExpired = new Date();
+  const mockRouteParams = {
+    orgId: 'acme',
```

Comment:
This is more of a nit, but we generally create org fixtures using `TestStubs.Organization()` which has a default slug of `org-slug` - this is less of an issue here since we don't use an org.

Reply:
Makes sense to me -- changed